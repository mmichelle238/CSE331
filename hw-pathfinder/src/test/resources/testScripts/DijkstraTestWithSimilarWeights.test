# Find a path with Dijkstra's algorithm when
# the path found by BFS is different. The weights are also
# very similar.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 3.456
AddEdge simpleGraph A C 1.534
AddEdge simpleGraph C D 1.563
AddEdge simpleGraph D B 0.545

FindPath simpleGraph A B