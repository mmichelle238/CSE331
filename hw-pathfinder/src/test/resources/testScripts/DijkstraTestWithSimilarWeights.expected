# Find a path with Dijkstra's algorithm when
# the path found by BFS is different. The weights are also
# very similar.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 3.456 from A to B in simpleGraph
added edge 1.534 from A to C in simpleGraph
added edge 1.563 from C to D in simpleGraph
added edge 0.545 from D to B in simpleGraph

path from A to B:
A to B with weight 3.456
total cost: 3.456